src/material-experimental/mdc-form-field/_form-field-theme.scss

```diff
@@ -72,14 +72,30 @@
     @include mdc-line-ripple.core-styles($query: mdc-helpers.$mat-typography-styles-query);
     @include form-field-subscript.private-form-field-subscript-typography($config);
 
-    // MDC uses the `subtitle1` level for the input label and value, but the spec shows `body1` as
-    // the correct level.
-    .mat-mdc-form-field-control,
-    .mat-mdc-form-field label,
-    .mat-mdc-form-field-text-prefix,
-    .mat-mdc-form-field-text-suffix {
+    // MDC uses `subtitle1` for the input value, placeholder and floating label. The spec
+    // shows `body1` for text fields though, so we manually override the typography.
+    // Note: Form controls inherit the typography from the parent form field.
+    .mat-mdc-form-field,
```

Comment:
I think changing these selectors broke some of the typography. I'm seeing a few screenshots in g3 where the font changed. Can we change them back?